include/fastdds/statistics/dds/domain/DomainParticipant.hpp

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file DomainParticipant.hpp
+ *
+ */
+
+#ifndef _STATISTICS_DDS_DOMAIN_DOMAINPARTICIPANT_HPP_
```

Comment:
The following suggestion applies to all public headers. At least for new files, we should mimic the full path

```suggestion
#ifndef _FASTDDS_STATISTICS_DDS_DOMAIN_DOMAINPARTICIPANT_HPP_
```